src/pal/src/safecrt/sscanf.c

```diff
@@ -20,7 +20,7 @@
 #include "mbusafecrt_internal.h"
 
 typedef int (*INPUTFN)(miniFILE *, const unsigned char*, va_list);
```

Comment:
It should already be correct signature. I thought only wide char needed consistency.